### PR TITLE
*: add builtin function UTC_TIMESTAMP

### PR DESCRIPTION
**Pull request by zyguan**
Hi, @shenli @hanfei1991 @tiancaiamao !
I just added the UTC_TIMESTAMP function, PTAL :)

### CODE REVIEW

**Comment by shenli** [comment]
@zyguan Good job!
@tiancaiamao @XuHuaiyu PTAL

File: parser/parser_test.go [hunk 1]
```diff
@@ -584,6 +584,11 @@ func (s *testParserSuite) TestBuiltin(c *C) {
 		{"select curtime()", true},
 		{"select curtime(6)", true},
 
+		// select utc_timestamp
+		{"select utc_timestamp", false},
```
Comment (shenli) [inline parser/parser_test.go]: This is valid in mysql:
mysql> select utc_timestamp;
+---------------------+
| utc_timestamp       |
+---------------------+
| 2017-02-05 02:10:07 |
+---------------------+
1 row in set (0.01 sec)

**Comment by tiancaiamao** [comment]
LGTM

**Comment by coocood** [comment]
LGTM
@zyguan 
Thank you!
Please resolve conflict.

**Comment by ngaut** [comment]
Good job! @zyguan

**Pull request merged**